env.gpu.yml

```diff
@@ -5,7 +5,7 @@ channels:
 dependencies:
   - cudatoolkit=10.2
```

Comment:
Thoughts on upgrading this?

Reply:
I've been using 11.1 with PyG 2.0.3 and pytorch 1.9.0. Let me try out CUDA 11.3 with pytorch 1.10.1, PyG 2.0.4.